cmd/archeio/app/buckets.go

```diff
@@ -94,7 +94,7 @@ type cachedBlobChecker struct {
 	blobCache
 }
 
-func newCachedBlobChecker() *cachedBlobChecker {
+func NewCachedBlobChecker() BlobChecker {
```

Comment:
either way, let's keep exported APIs under pkg/

Reply:
Agreed; the changes here are pretty small (just making some things public, and no actual code changes), and I wanted to get agreement before starting to move code around!